stable/grafana/values.yaml

```diff
@@ -197,11 +197,13 @@ persistence:
   # subPath: ""
   # existingClaim:
 
+v3io: {}
+
 initChownData:
   ## If false, data ownership will not be reset at startup
   ## This allows the prometheus-server to be run with an arbitrary user
   ##
-  enabled: true
+  enabled: false
```

Comment:
Suggest to leave defaults as they were (which made everything work in the default config) and not change them. So you'll have minimum changes from upstream, and override with values from provazio, which you're gonna have to change anyways.
That way with minimal changes it will be easy to maintain, upgrade and PR to upstream